lib/Github/Api/PullRequest/Review.php

```diff
@@ -159,11 +159,22 @@ public function submit($username, $repository, $pullRequest, $id, array $params
      * @param string $repository  the repository
      * @param int    $pullRequest the pull request number
      * @param int    $id          the review id
+     * @param string  $message    a mandatory dismissal message
      *
      * @return array|string
      */
-    public function dismiss($username, $repository, $pullRequest, $id)
+    public function dismiss($username, $repository, $pullRequest, $id, $message)
```

Comment:
If the message is not provided it seem that Github does not like it very well. I was expecting something like a 422 Error, but I got a 500 instead (A ruby nil error seem to be uncaught by Github).